dicomIndex.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>NCG</title>
+</head>
+
+<body>
+  <div id="app"></div>
+  <script type="text/javascript" src="/bundle.js"></script>
+  </link>
+  <script src="https://unpkg.com/@ohif/viewer@3.7.4/dist/index.umd.js" crossorigin></script>
+  <script>
+    var containerId = "root";
+    var componentRenderedOrUpdatedCallback = function () {
+      console.log('OHIF Viewer rendered/updated');
+    }
+
+    window.config = {
+      showStudyList: false,
+      servers: {
+        dicomWeb: [
+          {
+            name: 'Orthanc',
+            wadoUriRoot: 'wado',
+            qidoRoot: 'https://ncg-dev.projecteka.in/dicom-server',
```

Comment:
instead of making it fixed, can you pass these through from the caller? we can't have them fixed here. they ought to be dynamic. we can either rely on react or redux state or other way like LocalStorage (where we pass a key to identify such configurations).